Fix_git_init/Fix_git_init_home_directory.md

```diff
@@ -0,0 +1,31 @@
+﻿## **How to fix git init on home directory**

```

Comment:
What is the clear motivation for that?
I don't really see the need for that ... 
@bcharlier wdyt?

Reply:
It's just a fairly recurring error that Mr. Charlier has warned us about and I think it's relevant to know how to correct/fix it

Reply:
Then, I can conceive that it's not that important after all